CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Backpack changelog
 
 [Unreleased changes](./UNRELEASED.md).
+# 2019-06-17 - `BpkText` new tagName option `text`
+
+**Changed:**
+- bpk-component-text: 1.0.130 => 1.0.131
+  - Added new supported tagName option `text` for using Backpack text component within SVG elements.
```

Comment:
Could you move this to `UNRELEASED.md` and remove the version number? We typically add changes there and then move them to the changelog and add the version number after the release.

Reply:
Yeah sure thing!

Reply:
Updated